assets/src/components/editing/elements/image/block/ImageElement.tsx

```diff
@@ -10,6 +10,16 @@ import { ImagePlaceholder } from 'components/editing/elements/image/block/ImageP
 import { ImageSettings } from 'components/editing/elements/image/ImageSettings';
 
 interface Props extends EditorProps<ContentModel.ImageBlock> {}
+
+const cssWidth = (width: number | string | undefined) => {
+  if (!width) return undefined;
+  if (typeof width === 'number') return width; // Numbers are fine, they imply pixels
+
+  const pixels = parseInt(width, 10);
```

Comment:
do we not allow `50%` ?

Reply:
oh! I don't know. Do we allow percentages and/or units?

I guess either of these could be valid:

50%
100px
10em
2vw

I'm going to go fix that.

Reply:
The phoenix/elixir side only does numbers and percent, so I updated this to match.